src/components/Player/Pages/Activity/ActivityCalendar.jsx

```diff
@@ -43,7 +47,7 @@ const getNormalizationRatio = (data) => {
 const getTooltip = (date, data) => {
   const dateString =
     date &&
-    date.toLocaleDateString('en-US', {
+    date.toLocaleDateString(langCode, {
```

Comment:
what happens if we pass a language code that the browser doesn't recognize? Should we fallback to en-us?

Reply:
passing an unknown language code will not cause any issues. the parameter is optional and it uses the browser's language preference by default.